app/lib/actions/katello/content_view/add_rolling_repo_clone.rb

```diff
@@ -29,7 +29,8 @@ def plan(content_view, repository_ids)
         def run
           if Setting[:foreman_proxy_content_auto_sync]
```

Comment:
While I am in the business of not planning unnecessary things, would it make sense to pull this `if Setting[:foreman_proxy_content_auto_sync]` condition up around the `plan_self(repository_ids: clone_ids)` in line 26?

i.e. If the setting is not set just don't even plan the run phase.